src/main.rs

```diff
@@ -36,6 +37,7 @@ fn main() {
         ],
     );
 
-    let server = SpaceapiServer::new(host, status);
+    let datastore = datastore::RedisStore::new().unwrap();
+    let server = SpaceapiServer::new(host, status, datastore);
```

Comment:
@rnestler here I'm stuck. This would move `datastore` to the `SpaceServer`. The problem is, that for that, it needs to implement `Send` (be transferrable between threads) and `Sync` (be thread-safe).

In this case it's fine that the datastore is moved. There should only be one reference to it.

What do we need to do to get this to work? Implement Sync / Send / Copy for the DataStore? Can we even do that?

Alternatively, maybe we need to use something like Box, Rc or Arc?